helpdesk/serializers.py

```diff
@@ -70,8 +73,19 @@ def get_row_class(self, obj):
 
     def get_kbitem(self, obj):
         return obj.kbitem.title if obj.kbitem else ""
-
-
+        
+    def get_last_followup(self, obj):
+        followup = obj.followup_set.order_by().annotate(
+            last_followup=Window(
+                expression=Max("date"),
+                partition_by=[F("ticket_id"),],
+                order_by="-date"
+            )
+        ).values("last_followup").distinct().values_list("last_followup", flat=True)
+        # If there are no followups for the ticket then the result will be empty
+        return localtime(followup[0]).strftime('%Y-%m-%d %H:%M:%S') if followup else ""
```

Comment:
Actually there is no need for this lookup at all since the primary query already does the lookup.
Just this will work and no wasting another DB lookup:
```
    def get_last_followup(self, obj):
        return obj.last_followup
```